src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -588,9 +635,18 @@ private int decodeOpen(
 
         if (open != null)
```

Comment:
If the `else` clause is expected to be impossible due to earlier decode frame header logic, then we should `assert open != null` instead, same for other performatives.